src/IO/ITKImageInputFileFormat.cxx

```diff
@@ -336,46 +372,55 @@ read_file_itk(const std::string &filename)
 }
 
 template<typename ITKImageType>
-void
-orient_ITK_image(typename ITKImageType::Pointer &itk_image,
-                 CartesianCoordinate3D<float> &voxel_size,
-                 BasicCoordinate<3,int> &min_indices,
-                 BasicCoordinate<3,int> &max_indices,
-                 CartesianCoordinate3D<float> &origin,
+typename ITKImageType::Pointer
+orient_ITK_image(shared_ptr<ExamInfo> exam_info_sptr,
                  const typename ITKImageType::Pointer itk_image_orig)
 {
+  typedef itk::OrientImageFilter<ITKImageType,ITKImageType> OrienterType;
+  typename OrienterType::Pointer orienter = OrienterType::New();
+  orienter->UseImageDirectionOn();
+  orienter->SetInput(itk_image_orig);
+
+  // We need the origin to be in the minimum x, y, z corner. This
+  // depends on the patient position
+  switch (exam_info_sptr->patient_position.get_position()) {
+  case PatientPosition::unknown_position:
+    // If unknown, assume HFS
+    // TODO: warning?
```

Comment:
yes please.

Reply:
Actually, best when we set patient_position to unknown I think